src/gql/fragments/projectSettings/projectEventSettings.graphql

```diff
@@ -12,7 +12,6 @@
 
 fragment projectEventSettings on ProjectEventSettings {
   projectRef {
-    id
```

Comment:
The cache is probably overwriting the values since it thinks they are the same object.

Reply:
In case you actually do end up needing the id field the best solution would be to add a custom merge policy for this case. https://www.apollographql.com/docs/react/caching/cache-field-behavior